.drone.env

```diff
@@ -1,3 +1,3 @@
 # The test runner source for UI tests
-WEB_COMMITID=cb15d9035ec81560444caa7ba14f41ba840d64ff
-WEB_BRANCH=master
+WEB_COMMITID=74da0b360f445b14a840a313f12378cdfad7cad0
+WEB_BRANCH=stable-7.0
```

Comment:
@ScharfViktor @individual-it  need you advice or at least attention here. We already have a `stable-7.0` branch in web and only accept necessary bugfixes on that branch. Procedure is the same as with the `stable-6.0` branch half a year ago. I see no other way than switching the ocis CI to the stable branch (because the web assets come from there). Also means that further test work needs to appear on `stable-7.0` as well (ideally working against stable-7.0 and forward port to master on a regular basis). What do you think?

Reply:
I think we can go back to using the stable branch, I'm fine with that

Reply:
me too

Reply:
thank you guys!